sdk/test/trace/tracer_test.cc

```diff
@@ -57,6 +57,15 @@ std::shared_ptr<opentelemetry::trace::Tracer> initTracer(
   std::shared_ptr<SimpleSpanProcessor> processor(new SimpleSpanProcessor(std::move(exporter)));
   return std::shared_ptr<opentelemetry::trace::Tracer>(new Tracer(processor));
 }
+
+std::shared_ptr<opentelemetry::trace::Tracer> initTracer(
+    std::shared_ptr<std::vector<std::unique_ptr<SpanData>>> &received,
+    std::shared_ptr<Sampler> sampler)
+{
+  std::unique_ptr<SpanExporter> exporter(new MockSpanExporter(received));
```

Comment:
nit: use `make_unique` and `make_shared` instead of `new` to create objects:
`auto exporter = std::make_unique<MockSpanExporter>(received)`

Reply:
Changed `new` to `make_shared`. Since `make_unique` is only available since C++14, I left the `unique_ptr` part unchanged.